components/Cards/examples/EditReorderCollectionViewController.swift

```diff
@@ -116,22 +128,36 @@ class EditReorderCollectionViewController: UIViewController,
 
   func collectionView(_ collectionView: UICollectionView,
                       cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-    let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell",
-                                                  for: indexPath) as! MDCCardCollectionCell
 
-    MDCCardThemer.applyScheme(cardScheme, toCardCell: cell)
-    cell.isSelectable = (toggle == .edit)
+    let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell", for: indexPath)
+    guard let cardCell = cell as? CardEditReorderCollectionCell else { return cell }
+
+    MDCCardThemer.applyScheme(cardScheme, toCardCell: cardCell)
+
+    let title = dataSource[indexPath.item].title
+    let imageName = dataSource[indexPath.item].image
+    cardCell.configure(title: title, imageName: imageName)
```

Comment:
I believe you need to set the cardCell's `typographyScheme` before configuring it. Otherwise it might use an incorrect scheme.